packages/atomic/cypress/integration/search-box.cypress.ts

```diff
@@ -32,18 +30,16 @@ describe('Search Box Test Suites', () => {
   }
 
   describe('default search box', () => {
-    const htmlCode = '<atomic-search-box></atomic-search-box>';
     beforeEach(() => {
-      setUpPage(htmlCode);
-      generateAliasForSearchBox();
+      new TestFixture().with(addSearchBox()).withAlias(searchBoxAlias()).init();
```

Comment:
What is `searchBoxAlias` used for? and should `addSearchBox` include the alias, or is there an advantage of keeping these steps separate?

Reply:
I separated them because otherwise, the "creation of alias for search box" is kind of opaque when you write the test and if you are not already familiar with it.


Seeing that a test `addSearchBox()` then that the test tries to look for `@searchBoxFirstDiv` alias, without explicitly seeing anything that `adds an alias` function felt weird 

Said differently, a newcomer would ask: Where did the alias came from in the first place?

Reply:
okay, sounds good